entries/data.xml

```diff
@@ -88,6 +89,7 @@ alert( $( "body" ).data( "foo" ) ); // undefined
 $( "body" ).data( "bar", "foobar" );
 alert( $( "body" ).data( "bar" ) ); // foobar
       </code></pre>
+      <p>jQuery 3 changes the behavior of this method to align it to the <a href="http://www.w3.org/TR/html5/dom.html#dom-dataset">Dataset API specifications</a>. Specifically, jQuery 3 transforms all the properties' key to be camelCase. Writing a statement like <code>$( "body" ).data( {"my-name": "aValue"} ).data();</code> will return <code>{ myName: "aValue" }</code>.</p>
```

Comment:
Mostly we wrap important version differences in `<strong>` tags (or at least the version).

> Dataset API specifications

Not sure myself, should this be "specification" or "specifications"